### DISCUSSION TITLE
vite:5.4.9,"sass": "^1.80.2" 启动报错，怎么解决

### DISCUSSION THREAD

**Discussion by Steeld2022** [element-plus/element-plus#18591] [Q&A]
![image](https://github.com/user-attachments/assets/4ce473dc-8fac-4b12-83bf-d01ab73eb8bb)

**Comment by warmthsea** [comment]
降低sass版本或者 https://github.com/element-plus/element-plus/pull/18579#issuecomment-2418911488
